test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/random/PostcodeGenerator.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.RandomStringUtils.random;
+
+public class PostcodeGenerator implements Generator<String> {
```

Comment:
This will generate some invalid postcodes. Again, probably shouldn't be in the framework.